sympy/printing/tests/test_mathml.py

```diff
@@ -561,16 +561,14 @@ def test_presentation_mathml_limits():
 
 
 def test_presentation_mathml_integrals():
-    integrand = x
-    mml_1 = mpp._print(Integral(integrand, (x, 0, 1)))
-    assert mml_1.childNodes[0].nodeName == 'msubsup'
-    assert len(mml_1.childNodes[0].childNodes) == 3
-    assert mml_1.childNodes[0].childNodes[0
-        ].childNodes[0].nodeValue == '&int;'
-    assert mml_1.childNodes[0].childNodes[1
-        ].childNodes[0].nodeValue == '0'
-    assert mml_1.childNodes[0].childNodes[2
-        ].childNodes[0].nodeValue == '1'
+    assert mpp.doprint(Integral(x, (x, 0, 1))) == '<mrow><msubsup><mo>&#x222B;</mo><mn>0</mn><mn>1</mn></msubsup><mi>x</mi><mo>&dd;</mo><mi>x</mi></mrow>'
```

Comment:
For these long lines we could do something like this in the future:
```python
def lines(s, c=64):
    out = []
    assert '"' not in s
    while s:
        out.append(s[:c])
        s=s[c:]
    return '("%s")' % '"\n"'.join(out)
>>> lines('this is a really long line that I do not care where it breaks, I just want shorter lines')
("this is a really long line that I do not care where it breaks, I"
" just want shorter lines")
>>> print(("this is a really long line that I do not care where it breaks, I"
" just want shorter lines"))
this is a really long line that I do not care where it breaks, I just want shorter lines
```

Reply:
Ahh, thanks! More of those in #16214... But I can give everything a shortening at some stage.